x-pack/plugins/security_solution/public/detections/components/callouts/missing_privileges_callout/translations.tsx

```diff
@@ -140,6 +140,19 @@ const getPrivilegesExplanation = (missingPrivileges: string[], index: string) =>
     .join(' ');
 };
 
+const missingPrivilegesMessage = (index: string, privileges: string[]) => {
```

Comment:
nit: Interesting an input parameter is `index` while it can be a data stream name as well. I'm not sure if we have some generic name for indices and data streams but `indexOrDataStreamName` could be more explanatory here.

'missingPrivileges` could be also much more verbose than just `privileges`.

Reply:
I been puzzled with the same thought.

https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html, essentially data stream is time series data across multiple indices. So, technically index can be applied in this case and it's not wrong. Since, index in that case can be alias as well, again multiple underlying indices.

> 'missingPrivilegescould be also much more verbose than justprivileges`.

In this case, I just followed naming convention within this component